app/services/device_tracking/list_device_events.rb

```diff
@@ -0,0 +1,9 @@
+module DeviceTracking
+  class ListDeviceEvents
+    def self.call(user, device_id)
+      user_id = user.id
+      return [] unless Device.where(user_id: user_id, device_id: device_id)
+      Event.where(user_id: user_id, device_id: device_id).order(created_at: :desc)
```

Comment:
This list might get a little polluted over time. How do you feel about a limit here?

Reply:
Yes definitely needs a limit on devices.  I put an index in for that purpose.  I'll add...

Reply:
I meant events...